<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Запрос:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="queryInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeSelector">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1920x1080</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1680x1050</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1600x900</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1440x900</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1366x768</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280x1024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1024x768</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Разрешение экрана:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
